example_test.go

```diff
@@ -0,0 +1,43 @@
+package wazero
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+)
+
+// This is an example of how to use WebAssembly via adding two numbers.
```

Comment:
Ex this is intentional overlap with the examples directory, only to seed one example into https://pkg.go.dev/github.com/tetratelabs/wazero